### ISSUE TITLE
AssemblyScript Public Meeting #39 - June 16th 2021

### ISSUE THREAD

**Issue by torch2424** [AssemblyScript/core#65]
# AssemblyScript Public Meetings

Anyone who is contributing to the AssemblyScript project, building something with AssemblyScript, interested in the Assembly project or WebAssembly in general, is welcome to join! 😄

This meeting is focused on both fostering feedback from the AssemblyScript communitty and ecosystem, as well as for maintainers to discuss technical problems or achievements on AssemblyScript projects. :rocket: 

# Date and Time

AssemblyScript Public meetings occur every two weeks.

This public meeting will take place: **June 16th, 2021, 17:00 UTC (10:00 AM US PT, UTC -8)**

# General Agenda

* Agenda Items from comments left on this Github issue
  * **Please feel free to comment below to add anything to the agenda.**
* Additional in-meeting comments / discussion

Feel free to comment on this issue if you have any agenda items you would like to bring up. 
Meeting Notes will be placed on this issue for those who cannot make the meeting.

# Meeting Information

We will use the AssemblyScript discord for meetings: https://discord.gg/vSBbJ5T . We will meet in the "meeting" voice channel :+1: 

Notes will be taken by the host(s) ( @torch2424 and/or @willemneal ), and posted after the meeting. Notes are free to be edited through comments on the meeting notes at a later time.

**Comment by torch2424** [comment]
A few things I wanted to bring up:

* Can we cut a new path release for: https://github.com/AssemblyScript/assemblyscript/pull/1894 
* Devrel: Various discussions I've been having with folks trying out AS 😄 
* Latest discussions around string encodings [[1](https://github.com/WebAssembly/interface-types/issues/136), [2](https://github.com/WebAssembly/design/issues/1419), [3](https://github.com/WebAssembly/interface-types/issues/135)]

**Comment by torch2424** [comment]
## Meeting Notes

* @torch2424 : can I cut a release
   * Prepend fix: or feat: to a commit, and then it will automatically cut a release
      * Cool, I’ll open a PR after this to do that
* @torch2424 : Been doing devrel discussions
   * Some on the Fastly side, that I don’t know if I can share
   * One I definitely can: Polythene, (Polyphene.io redirects to https://holium.org/ )
      * They are looking into using AS as a supported language
* @torch2424 : How are the string encodings discussions going?
   * @battlelinegames & @dcodeIO : Things are going well! We think
      * Discussions with Luke seem cool, we got UTF-16 supported, but seems like we got some push back from Andreas Rossberg.
      * (More discussions about the current state of spec discussions)
* @jtenner : As-covers
   * It’s working! And works on the compiler!
      * Compiler has 54% coverage
   * Huge shoutout to @JairusSW for all of their amazing work!
   * Working on getting code coverage support into as-pect
* @jtenner : as-lunatic
   * Has a new version, improves it’s package handling! 
   * Passes TCP test suite.
* @jtenner : Adding blob to the stdlib?
   * https://developer.mozilla.org/en-US/docs/Web/API/Blob 
   * Already have a prototype, but following the Node Blob
   * @dcodeIO : Shouldn’t be too hard to implement, but maybe we should skip on that for a while.
      * Once we have stack switching and things, we can start an implementation on promises, and then we can better match the api
* @dcodeIO : What happened to the as-node package?
   * @jtenner : I’ve been waiting for WASI to mature before starting on the filesystem
      * Gnarly preopens logic
   * @jtenner & @torch2424 : Other classes weird or impossible to implement:
      * Buffers are weird, event emitters are not possible, etc…
         * @JairusSW : Event emitters are possible: https://github.com/aspkg/as-events/blob/master/assembly/index.ts

**Comment by torch2424** [comment]
Closing this in favor of: https://github.com/AssemblyScript/core/issues/66 , feel free to continue discussions from this meeting here. 😄
